### PR TITLE
Landing page of the ecommerce screen set up

### PR DESCRIPTION
**Pull request by dev-enforced**
This pull request has :
The home page layout set up (with navbar,advantages section, featured products section ,footer)
Has utility classes attached to the base file.

Preview Link: https://deploy-preview-2--chillar-party.netlify.app/

Preview Image: 
image
image
image

### CODE REVIEW

**Review by krishnakant01 — APPROVED** [review]
code looks clean. One UI suggestion, don't use red color for text in search bar

**Comment by dev-enforced** [comment]
Sure @krishnakant01 making the changes!!Thanks for the suggestion

File: styles/home-page.css [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+}
+
+.whole-content-container{
+    display: flex;
+    flex-direction: column;
+    height: 100vh;
+}
+
+.main-content-container{
+    flex-grow: 1;
```
Comment (rkapoor10) [inline styles/home-page.css]: define this in base css

File: styles/home-page.css [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+}
+
+.image-banner{
+    background-image: url("../assets/images/brand-items.png");
+    background-size: auto;
+    background-repeat: no-repeat;
+    background-position: center center;
+    height:40rem;
+    display: flex;
```
Comment (rkapoor10) [inline styles/home-page.css]: you already have flex in base css

File: styles/home-page.css [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+    width: 100%;
+}
+
+.banner-content{
+    display: flex;
+    flex-flow:column wrap;
```
Comment (rkapoor10) [inline styles/home-page.css]: same as above

File: styles/home-page.css [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+.banner-title{
+    font-family: 'Amatic SC',cursive;
+    font-weight: var(--font-bold);
+    color: var(--ecomm-txt);
+}
+
+.ecom-sections{
+    background-color: var(--ecomm-secondary);
+    display: flex;
+    flex-direction: column;
```
Comment (rkapoor10) [inline styles/home-page.css]: same as above

**Review by rkapoor10 — APPROVED** [review]
The code looks good to merge. Suggested a few changes you may go through them.

File: styles/home-page.css (hunk 4 above)
Comment (dev-enforced) [reply styles/home-page.css]: Thanks @rkapoor10 for pointing to the utility classes and using it appropriately in the HTML

**Pull request merged**